index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Figma to HTML & CSS </title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Inter:wght@500;800&family=Oooh+Baby&family=Yanone+Kaffeesatz:wght@300&display=swap"
+        rel="stylesheet">
+</head>
+
+<body>
+    <div class="app">
+        <div class="heroContainer">
+
+            <nav class="nav-bar">
+
+                <ul class="nav-bar-list">
+                    <image class="logo" src="images/Landifylogo.png"></image>
+                    <li><a class="nav-link-text" href="#">About</a></li>
+                    <li><a class="nav-link-text" href="#">Products</a></li>
+                    <li><a class="nav-link-text" href="#">Pricing</a></li>
+                    <li><a class="nav-link-text" href="#">Blog</a></li>
+                    <li><a class="nav-link-text" href="#">Jobs</a></li>
+                    <li><a class="nav-link-text" href="#">More</a></li>
+                </ul>
+
+                <div class="nav-image-list">
+
+                    <img src="Images/Google Play Badge.png" alt="Google playstore">
+                    <img src="Images/App Store Badge.png" alt="App store logo">
+                </div>
+
+
+            </nav>
+            <section class="heroContent">
+                <div class="content">
+                    <div class="heroTitle">
+
+                        <h1>Organize Products.<br> Get more done.<br>
+                            <button class="start-button">Get Started</button>
+                        </h1>
+                        <img class="phone-image" src="images/iPhone X.png" , alt="image of a phone">
```

Comment:
- [ ] Remove the "," in the img tag. its irrelevant here.
- [ ] Make sure to add files without any spaces in them. Use kebab-case for naming them